Xamarin.Forms.Core/Shell/ShellSection.cs

```diff
@@ -1013,13 +1013,13 @@ void RemovePage(Page page)
 
 		void SendAppearanceChanged() => ((IShellController)Parent?.Parent)?.AppearanceChanged(this, false);
 
-		void SendUpdateCurrentState(ShellNavigationSource source)
-		{
-			if (Parent?.Parent is IShellController shell)
-			{
-				shell.UpdateCurrentState(source);
-			}
-		}
+		//void SendUpdateCurrentState(ShellNavigationSource source)
```

Comment:
Can we remove this code? and the other commented code?

Reply:
👍